parameter_test.go

```diff
@@ -0,0 +1,285 @@
+package tfe
+
+import (
+	"context"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestParametersList(t *testing.T) {
+	client := testClient(t)
+	ctx := context.Background()
+
+	orgTest, orgTestCleanup := createOrganization(t, client)
+	defer orgTestCleanup()
+
+	psTest, _ := createPolicySet(t, client, orgTest, nil, nil)
+
+	vTest1, _ := createParameter(t, client, psTest)
+	vTest2, _ := createParameter(t, client, psTest)
+
+	t.Run("without list options", func(t *testing.T) {
+		vl, err := client.Parameters.List(ctx, psTest.ID, ParameterListOptions{})
```

Comment:
```suggestion
		pl, err := client.Parameters.List(ctx, psTest.ID, ParameterListOptions{})
```

Goes for the whole file of course.